x/nameservice/04_events.md

```diff
@@ -0,0 +1,39 @@
+<!--
+order: 4
+-->
+
+# Events
+
+The `x/nameservice` module emits the following events:
+<!-- placeholder, must learn more about events -->
```

Comment:
go events and messages are new terrain for me, so I staged this topic and am now asking questions